helper/resource/testing_new.go

```diff
@@ -117,6 +117,9 @@ func runNewTest(t testing.T, c TestCase, helper *tftest.Helper) {
 					t.Fatalf("Step %d/%d error running import, expected an error with pattern (%s), no match on: %s", i+1, len(c.Steps), step.ExpectError.String(), err)
 				}
 			} else {
+				if err != nil && c.ErrorCheck != nil {
+					c.ErrorCheck(err)
```

Comment:
should this have a way to say ignore the error? like a return of `(bool, error)` or something? so this could be an alternative `ExpectError` but let you use things like `errors.Is` / `errors.As` and flag it as ok for the test?

just throwing this out, not sure if it makes sense

Reply:
I can't foresee a use for that which, of course, means someone will need that functionality within 14ms of merging. I'll make a change.

Reply:
Updated